### ISSUE TITLE
/install/index.php stuck at "Processing..."

### ISSUE THREAD

**Issue by XDmToter** [WWBN/AVideo-Encoder#542]
I downloaded the latest release (3.7) of AVideo-Encoder and I am attempting to install.

All checks on /install/index.php pass.  I completed the form with all necessary entries.

When I click the "Install now" button, I get a popup with "Processing...", but it never goes away.  It's been stuck here for Hours.

I see the following in the PHP Log:

`PHP Fatal error:  Uncaught Error: Failed opening required '/encoder.example.com/public_html/view/../videos/configuration.php' (include_path='.:/opt/remi/php82/root/usr/share/pear:/opt/remi/php82/root/usr/share/php:/usr/share/pear:/usr/share/php') in /encoder.example.com/public_html/view/index.php:18`

I don't know if this is relevant, but it's the only Fatal error in the PHP logs.

The install script creates the "videos" directory, but it's empty.  No "configuration.php" file is ever written.  I also noticed that the database still doesn't have any tables.

**Comment by DanielnetoDotCom** [comment]
make sure you are using a database user and password with rights to create table

check your apache error log

**Comment by XDmToter** [comment]
Yes.  DB credentials are correct.  Nothing in the Apache error logs.  Just a single GET request in the access log when I load `/install/index.php`.  Interestingly, there is no POST shown in the access log when I click "Install now" and the "Processing..." message appears.

**Comment by DanielnetoDotCom** [comment]
check your JS console then, must be something in the logs, otherwise I cannot help

**Comment by XDmToter** [comment]
Thank you for the tip.   There was a CORS error when the Encoder setup script attempted to log in to the main site.  I had to remove the following from `.htaccess` on the main site:
```
<IfModule mod_headers.c>
    Header set Access-Control-Allow-Origin: *
</IfModule>
```

**Comment by DanielnetoDotCom** [comment]
I guess it is some configuration in your server, I do not have this issue on my side. 

In future updates you may need to overwrite this file again
